### PR TITLE
User can edit an entry - Cathy Garcia

### PR DESCRIPTION
**Pull request by cathyygrcia**
https://github.com/cathyygrcia/code-journal/assets/90016242/1ebafa7b-c63b-4923-9d63-61a67a98d905

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -96,3 +128,27 @@ const $newButton = document.querySelector('.new-button');
...
+const $entryNotes = document.querySelector('#notes');
+const $h1 = document.querySelector('h1');
+
+$ul.addEventListener('click', function pencilClick(event) {
+  if (event.target.tagName === 'I') {
+    const dataEntryId = event.target
+      .closest('li')
+      .getAttribute('data-entry-id');
+
+    viewSwap('entry-form');
```
Comment (shawnkost) [inline js/main.js]: I would call this after the loop, that way we swap after the inputs were filled in

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just need to fix the CSS change I messaged you about. The "Code Journal" text & the "Entries" text below it should be aligned. Left one other suggestion though

**Comment by cathyygrcia** [comment]
https://github.com/cathyygrcia/code-journal/assets/90016242/7cf3b8df-724c-4a9e-be9e-10d4046177d7

**Review by shawnkost — APPROVED** [review]
👍

**Pull request merged**